src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+        msg += "  Now you have " + list.size() + " task(s) in the list.";
+        return msg;
+    }
+    
+    public static String addDeadline(String line) {
+        String msg = "";
+        int ind1 = line.indexOf('/');
+        Task deadline = new Deadline(line.substring(9, ind1 - 1), line.substring(ind1 + 4));
+        list.add(deadline);
+        msg += "Got it. I've added this task: " + '\n';
+        msg += "    " + deadline.toString() + '\n';
+        msg += "  Now you have " + list.size() + " task(s) in the list.";
+        return msg;
+    }
+    
+    public static String addEvent(String line) {
+        String msg = "";
+        int ind1 = line.indexOf('/');
+        Task event = new Event(line.substring(6, ind1 - 1), line.substring(ind1 + 4));
+        list.add(event);
+        msg += "Got it. I've added this task: " + '\n';
+        msg += "    " + event.toString() + '\n';
+        msg += "  Now you have " + list.size() + " task(s) in the list.";
+        return msg;
+    }
+    
+    public static String list() {
+        String msg = "";
+        // accesses the list
+        if (list.size() == 0) {
+            msg += "list is empty";
+        } else {
+            msg += "Here are the tasks in your list:" + '\n';
+            int counter = 1;
+            for (Task s : list) {
+                msg += "    " + counter + ". " + s.toString();
+                counter++;
+                msg += '\n';
+            }
+        }
+        return msg;
+    } 
+    
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        String name = "**Rick**";
+        String line = "____________________________________________________________";
+        System.out.println(line);
+        System.out.println("  Hello! I'm " + name);
+        System.out.println("  What can I do for you?");
+        System.out.println(line + '\n');
+        while (true) {
+            String str = sc.nextLine();
+            String msg = "  ";
+            if (str.equals("bye")) {
+                // exits the program
+                break;
+            } else if (str.equals("list")) {
+                msg += list();
+            } else if (str.split(" ")[0].equals("done")) {
+                msg += makeDone(str);
+            } else if (str.split(" ")[0].equals("todo")) {
+                msg += addTodo(str);
+            } else if (str.split(" ")[0].equals("deadline")) {
+                msg += addDeadline(str);
+            } else if (str.split(" ")[0].equals("event")) {
+                msg += addEvent(str);
+            } else {
+                msg += "Unknown command";
+            }
```

Comment:
Perhaps you can consider using a switch here.

Reply:
In addition, I feel that you can store a the return value of str.spilt and str.equals into a String variable, so that it can be more clear. This combined with the switch method will make the code clearer. 

For example:
String something;
switch (something) {
case "something" {
//do something if something is done
break;
}